src/main/scala/io/picnicml/doddlemodel/preprocessing/QuantileDiscretizer.scala

```diff
@@ -0,0 +1,101 @@
+package io.picnicml.doddlemodel.preprocessing
+
+import breeze.linalg.DenseVector
+import breeze.stats.DescriptiveStats
+import io.picnicml.doddlemodel.CrossScalaCompat._
+import io.picnicml.doddlemodel.data.Feature.FeatureIndex
+import io.picnicml.doddlemodel.data.Features
+import io.picnicml.doddlemodel.syntax.OptionSyntax._
+import io.picnicml.doddlemodel.typeclasses.Transformer
+import scala.Double.{MinValue, MaxValue}
+
+case class QuantileDiscretizer(private val bucketCounts: DenseVector[Double], private val featureIndex: FeatureIndex, private val quantiles: Option[Seq[Seq[(Double, Double)]]] = None) {
```

Comment:
Please use
```scala
case class QuantileDiscretizer(private val bucketCounts: DenseVector[Double],
                               private val featureIndex: FeatureIndex,
                               private val quantiles: Option[Seq[Seq[(Double, Double)]]] = None) {
```
to make sure no line is longer than 120 chars. There are also some other lines longer than that.

Reply:
Sure - I'm so used to Scalafmt that I forgot this repo didn't have it!